css/css-grid/parsing/grid-auto-columns-valid.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
```

Comment:
I guess it'd be nice to have also the `grid-auto-rows` version. I guess implementations would share the parsing logic, but having a test covering the other property seems like a good idea.

Reply:
In css/css-backgrounds/parsing/border-radius-valid.html I test border-top-left-radius...border-bottom-left-radius as it is obvious where to look for the test.

For grid-auto-rows and grid-auto-columns, it seemed clearer to have separate files.

Reply:
Ok, I didn't realize we have two files to cover columns and rows. Sorry about that.